.gitignore

```diff
@@ -2,3 +2,4 @@
 __pycache__
 n2y.egg-info/
 .DS_Store
+build/
```

Comment:
Good idea

Reply:
Just made an addition at lines 38-39, is that what you meant?